bin/configs/jaxrs-spec-swagger-annotations.yaml

```diff
@@ -0,0 +1,11 @@
+generatorName: jaxrs-spec
+outputDir: samples/server/petstore/jaxrs-spec-swagger-annotations
```

Comment:
please add the folder to the CI: https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-jaxrs.yaml

so that it will be tested moved forward.

Reply:
update: done via https://github.com/OpenAPITools/openapi-generator/pull/22314